### ISSUE TITLE
Better water collection

### ISSUE THREAD

**Issue by DokimiCU** [jeremyshannon/Exile#131]
(I'm transferring old but still relevant issues from my unmaintained version)

"I've spent so much time and so many resources trying to get my water thing fired with no luck. Can't I dig a seepage hole? Leave it to bake in the sun? Use half a coconut? If I ever find a granite boulder I'll make a mortar, can I use that?
I appreciate the hard work, but there's a level of abstraction in not only the cubic nature of the world, but the understanding of the users that needs a bit more love."

Basically: ways to get water that don't use unfamiliar mechanics (e.g. pottery firing, seepage holes)

One aspect of issue #40

**Comment by Crystalwarrior** [comment]
Seepage holes are a great idea!
